### PR TITLE
[10.0][WIP] project recalculate hourly-based

### PR DESCRIPTION
**Pull request by sergiocorato**
Replace day based computation on task with hours based computation, with a optional module to install.

### CODE REVIEW

File: project_recalculate_hours/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Sergio Corato
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Project Recalculate Hours",
+    "version": "10.0.1.0.0",
+    "author": "Efatto.it, "
+              "Odoo Community Association (OCA)",
+    "website": "https://efatto.it",
```
Comment (nikul-serpentcs) [inline project_recalculate_hours/__manifest__.py]: @sergiocorato `https://github.com/OCA/project`

Comment (sergiocorato) [reply project_recalculate_hours/__manifest__.py]: done, thanks!

**Comment by pedrobaeza** [comment]
Please don't include plurals in module name.

**Comment by sergiocorato** [comment]
> Please don't include plurals in module name.

done, thanks!

**Comment by pedrobaeza** [comment]
Can you include a README for knowing the aim of this?

**Comment by sergiocorato** [comment]
> Can you include a README for knowing the aim of this?

done!

**Comment by rafaelbn** [comment]
Hi @sergiocorato , is still this PR WIP? If not please change tittle.

Also please review travis :red_circle: 

Thanks!!!!

**Pull request closed without merging**